template.html

```diff
@@ -197,7 +197,7 @@
           <div class="col-md-6 promo">
             <img class="logo" src="static/images/<= logo.nav =>.png" srcset="static/images/<= logo.nav =>@2x.png 2x"/>
             <p>
-              Etcher is an open source project by <a class="resin-link" href="https://resin.io">resin.io</a> - Modern DevOps for the Industrial Internet of Things
+              Etcher is an open-source project by <a class="resin-link" href="https://resin.io">resin.io</a> - Modern DevOps for the Industrial Internet of Things
```

Comment:
This hyphen should probably be left out, to match https://github.com/resin-io/etcher/blob/master/lib/gui/index.html#L44 ?